bin/travis_setup.sh

```diff
@@ -22,9 +22,11 @@ else
     sudo add-apt-repository --yes ppa:ubuntu-toolchain-r/test
     sudo apt-get -qq update
     sudo apt-get install -y -qq \
+      gcc-5 g++-5 \
```

Comment:
It is only for linux plarform.
GCC 4.9.0,	April 22, 2014
I agreed with you that it's very risky.
I found that scala-native doesn't work with java 1.7, it requires to upgrade to java 1.8 which is not supported on Ubuntu 14.04 for instance (there is third party package but it's not stable, shows errors)
I wasn't expected you accept this PR :) I just can't work on separate repository until scala-native becomes more mature.